doc/nrf/ug_nrf5340.rst

```diff
@@ -589,6 +591,42 @@ Additionally, the memory partitions must be defined and include:
 Samples and applications built for Thingy:53 enable simultaneous update of multiple images by default.
 To learn more about Thingy:53, see :ref:`ug_thingy53`.
 
+MCUboot's serial recovery of the networking core image
+******************************************************
+
+In addition to the recovery of the application core image, also the networking core image can be recovered.
+You can use this feature if you build nRF53 MCUboot with one of the following options:
+
+* :kconfig:option:`CONFIG_NRF53_MULTI_IMAGE_UPDATE` - Simultaneous multi-image DFU.
+* :kconfig:option:`CONFIG_NRF53_RECOVERY_NETWORK_CORE` - Serial recovery for the network core in the one image pair mode.
+
+To upload the networking image, use the following command::
+
+     ./mcumgr image upload <build_dir_path>/zephyr/net_core_app_update.bin -e -n 3 -c serial_conn
+
+``serial_conn`` is the serial connection configuration.
+
+To enable serial recovery of the network core while multi-image update is not enabled in the MCUboot, set the following options:
```

Comment:
Where can I find information about enabling serial recovery in the MCUboot docs?
I see we don't have much about this in the NCS docs either, so maybe we should adapt https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/applications/nrf_desktop/README.html#serial-recovery-dfu for the general docs? @MarekPieta , @pdunaj , opinions?

Reply:
Such a generic section would be definitely nice to have (maybe it could be even part of MCUboot's docs).
The docs of serial recovery in nRF Desktop could be used as a base (we would just need to remove references which are specific to nRF Desktop application).

Reply:
Maybe a section regarding serial recovery could fit in ug_fw_update.rst, but I see this as a separate task from this PR.